jujugui/static/gui/src/app/init.js

```diff
@@ -1512,7 +1512,18 @@ class GUIApp {
         if (error) {
```

Comment:
Let's convert entityId to an URL sooner rather than later. Two lines above this one, there is a weird check `if (entityId.indexOf('bundle/') > -1)` which could be rewritten as `if url.isBundle()`.